sdk/metric/controller/push/push.go

```diff
@@ -73,16 +73,21 @@ var _ Ticker = realTicker{}
 // exporter will be used as the label encoder for the SDK itself,
 // otherwise the SDK will be configured with the default label
 // encoder.
-func New(batcher export.Batcher, exporter export.Exporter, period time.Duration) *Controller {
+func New(batcher export.Batcher, exporter export.Exporter, period time.Duration, opts ...Option) *Controller {
```

Comment:
Should the documentation here be updated to reflect that this function now accepts `Option`s?

Reply:
Updated